### ISSUE TITLE
Some general UI Design notes

### ISSUE THREAD

**Issue by BigRoy** [ggarra13/mrv2#132]
I'm not a UI designer at all, but I just wanted to point some things out that stood out to me when using the UI. Feel free to ignore if you like it the current way for good reason.

Overall it feels like there's _a lot going on_ - largely due to the amount of borders on widgets.

![image](https://github.com/ggarra13/mrv2/assets/2439881/4dac5f64-5d5e-4187-a506-9274bcc6f64a)

I wonder if it'd still be clear if the Timeline and Top Bar both had no borders on them - likely it'd be easier on the eye.
The bigger border mania is the panels on the right hand side.

1. Top bar combobox/dropdown should have the rounded corners removed.

After clicking the comboboxes they seem contained with a rectangular box, but inside of it have rounded corners which makes it feel very cluttered.

![image](https://github.com/ggarra13/mrv2/assets/2439881/75fcce93-f208-47ce-8ab5-1cdd7ae9bc96)

Interestingly enough, when expanding the dropdowns the rounded corners disappear on the box itself. 🥳 But then suddenly the menus themselves have rounded corners?

![image](https://github.com/ggarra13/mrv2/assets/2439881/044e9ef3-9cde-40d8-b1ea-6147eaeebec8)

TL;DR avoid the rounded corners OR make sure the button always shows rounded only and the menus remain unaffected.

The rounded corners also somewhat clutter the sections in the panels (albeit not as bad as the dropdowns) here:

![image](https://github.com/ggarra13/mrv2/assets/2439881/944f8c59-4c85-4702-8289-2b339050b035)

2. Pixel bar has combination of font sizes in the comboboxes

The F and Y are bigger than the HSV and `x1.2`, see:

![image](https://github.com/ggarra13/mrv2/assets/2439881/cf339c2e-fe0c-445c-a4cc-8bea889673a9)

I'd expect the sizes to be consistent.

_Side note:_ If I click on the zoom button and e.g. set it to `9x` then the button shows `1x` directly after even though the view is zoomed. Moving around in the view does update it to 9x.

3. Blue line overlay over timeline obscures the frame tick indicator

Here's a screenshot of just the timeline.

![image](https://github.com/ggarra13/mrv2/assets/2439881/b141f9f7-e129-4174-8ad3-d98b034e27ce)

The blue and purple line (likely indicating the current memory cache?) cover so much of the vertical ticks across the timeline that you can hardly see the frames themselves.

Would it be possible to increase the height of the frame tick indicators so it's easier to identify roughly the frames in the timeline? It feels like currently there's only about two to three pixels above the two bars visible.

**Comment by ggarra13** [comment]
> 1. Top bar combobox/dropdown should have the rounded corners removed.

Done.

**Comment by ggarra13** [comment]
> 2. Pixel bar has combination of font sizes in the comboboxes

Done.

**Comment by ggarra13** [comment]
> _Side note:_ If I click on the zoom button and e.g. set it to `9x` then the button shows `1x` directly after even though the view is zoomed. Moving around in the view does update it to 9x.

Fixed.

**Comment by ggarra13** [comment]
> 3. Blue line overlay over timeline obscures the frame tick indicator

I'm going to send you to post the issue to https://github.com/darbyjohnston/tlRender as he is the one that controls the design of the timeline.

**Comment by ggarra13** [comment]
> The rounded corners also somewhat clutter the sections in the panels (albeit not as bad as the dropdowns) here:

What do you suggest?  Getting rid of the rounded corners?  I am particularly fond of them.

**Comment by ggarra13** [comment]
v1.1.0 has additional improvements to the resizing of the UI.
